src/dom/map-setup.ts

```diff
@@ -1,41 +1,48 @@
+import {
+  Circle,
+  LatLngExpression,
+  LayerGroup,
+  Map,
+  Polyline,
+  TileLayer,
+} from "leaflet";
+import { ILapDataset } from "../models/go-kart-types";
 import { lapTimer, stopTimer } from "../utility/timer";
 
-let map = L.map("map");
-const circles = L.layerGroup();
-let circle = L.circle();
+let map = new Map("map");
+const circles: LayerGroup = new LayerGroup([]);
+const initialLatLng: LatLngExpression = [-29.697911, 30.525229];
+let circle = new Circle(initialLatLng);
 
-const polylines = L.layerGroup();
-let latlngs = [];
+const polylines: LayerGroup = new LayerGroup([]);
+let latlngs: LatLngExpression[] = [];
```

Comment:
I'm pretty sure you can make the array const as you won't be reassigning the array it self but its content.

Reply:
@AyaazStreet there's a function that reassigns the array
```ts 
export function clearLatLngs() {
  latlngs = [];
}
```